python/optopy/nabla.py

```diff
@@ -15,27 +15,27 @@
 # to run execute: python -m unittest [-v] optopy.nabla
 class TestNablaFunction(unittest.TestCase):
     
-    def _get_nabla_op(self, dtype, dim):
+    def _get_nabla_op(self, dtype, dim, hx, hy, hz):
```

Comment:
Great work! However, I do not like the interface that hz is necessary for a 2d nabla operatpr

Reply:
Default arguments added. The 2d nabla op can be now called with `float_2d(hx, hy)` or `float_2d()`. The `_get_nabla_op` function is only used for unittesting.